backend/src/prompts/templates/best-tool.j2

```diff
@@ -0,0 +1,25 @@
+You are an expert at picking a tool to solve a task
+
+The task is as follows:
+
+{{ task }}
+
+below is your history of all work you have assigned and had completed by Agents
+Trust the information below completely (100% accurate)
+
+{{ scratchpad }}
+
+Pick 1 tool (no more than 1) from the list below to complete this task.
+Fit the correct parameters from the task to the tool arguments.
+If the parameters are tagged as optional, you do not need to fill them in,
+but feel free to if it is necessary
```

Comment:
I feel this wording can be improved to say that although optional you should try and fill in but do not make something up to fill it. (something along those lines)

Reply:
Updated